terraform/services/zscaler-security-groups/variables.tf

```diff
@@ -0,0 +1,27 @@
+variable "app" {
+  description = "The application name (ab2d, bcda, dpc)"
+  type        = string
+  validation {
+    condition     = contains(["ab2d", "bcda", "dpc"], var.app)
+    error_message = "Valid value for app is ab2d, bcda, or dpc."
+  }
+}
+
+variable "env" {
+  description = "The application environment (dev, test, mgmt, sbx, prod)"
+  type        = string
+  validation {
+    condition     = contains(["dev", "test", "mgmt", "sbx", "prod"], var.env)
+    error_message = "Valid value for env is dev, test, mgmt, sbx, or prod."
+  }
+}
+
+variable "vpc_id_public" {
+  description = "The vpc id public"
+  type        = string
+}
+
+variable "vpc_id_private" {
+  description = "The vpc id private"
+  type        = string
+}
```

Comment:
The vpc can be determined by the vpc module in main.tf. See how this is done at https://github.com/CMSgov/ab2d-bcda-dpc-platform/blob/0c40661080b9f0b49eef82237f6dbafecfc244b8/terraform/modules/function/main.tf#L181

Reply:
Wait, i don't think that's how it works. That's a vpc module, but we aren't creating a vpc we are referencing an existing one. So it's either an output.tf, from a remote state file , or an actual input.

Reply:
The vpc module gets us a reference to the existing VPC